  Fixed bug on remote exception: when it's an OException must be not wrapped  